ckan/tests/logic/action/test_get.py

```diff
@@ -1498,7 +1515,7 @@ def test_package_search_with_fq_for_create_user_id_will_include_drafts_for_other
 
         fq = "(creator_user_id:{0} AND +state:draft)".format(user['id'])
         results = logic.get_action('package_search')(
-            {'user': sysadmin['name']},
+            {'user': sysadmin['name'], 'sysadmin': True},
```

Comment:
There is no sense in new context parameters while they can be obtained from existing ones without overhead(`auth_user_obj`)